back/src/utils/logger.ts

```diff
@@ -167,17 +187,90 @@ export const createLogger = (filename: string): OpacifiedLogger => {
   };
 };
 
-const sanitizeEvents = (events: DomainEvent[]) =>
-  events.map(({ publications, id, topic, wasQuarantined }: DomainEvent) => {
-    const publishCount = publications.length;
-    const lastPublication = publications[publishCount - 1];
+const sanitize =
+  <T>(cb: (t: T) => Partial<T>) =>
+  (params: T | undefined) => {
+    if (!params) return;
+    return cb(params);
+  };
+
+const sanitizeError = sanitize<
+  Error | Partial<SQLError> | AxiosError<unknown, any>
+>((error) => {
+  if (isAxiosError(error))
+    return {
+      message: `Axios error : ${error.message}`,
+      response: sanitizeAxiosResponse(error.response),
+    };
+
+  return error;
+});
+
+const sanitizeAxiosResponse = sanitize<AxiosResponse>((response) => {
+  const req = response.config ?? response.request;
+
+  if (200 <= response.status && response.status < 400) {
```

Comment:
Possible de remettre à l'endroit ?
```suggestion
  if (response.status >= 200 && response.status < 400) {
```